### PR TITLE
Update Delton's information in AboutUs page

### LINKED ISSUE #33: Update Delton's AboutUs information

### PR DESCRIPTION
**Pull request by DeltonCheng**

### CODE REVIEW

**Review by leawang0 — APPROVED** [review]

File: docs/AboutUs.md [hunk 1]
```diff
@@ -9,15 +9,15 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
+### Delton Cheng
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/deltoncheng.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](https://github.com/DeltonCheng)]
+[[portfolio]()]
```
Comment (shuyangk) [inline docs/AboutUs.md]: Should you link the portfolio to your PPP file?

**Review by shuyangk — CHANGES_REQUESTED** [review]
Apart from the portfolio part, LGTM!

File: docs/AboutUs.md (hunk 1 above)
Comment (DeltonCheng) [reply docs/AboutUs.md]: Linked the PPP in #31, will deconflict when merging. Thanks for the reminder :)

Comment (DeltonCheng) [reply docs/AboutUs.md]: Ignore previous, have included the portfolio in this pull request as well. Thanks :)

**Review by shuyangk — APPROVED** [review]
LGTM!

**Pull request merged**